medical_patient_occupation/views/medical_patient_occupation_view.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+	<data>
+		<record id="medical_patient_occupation_view_tree" model="ir.ui.view">
+			<field name="name">medical.patient.occupation.tree</field>
+			<field name="model">medical.patient.occupation</field>
+			<field name="priority" eval="8" />
+			<field name="arch" type="xml">
+				<tree string="Patient Occupation">
+					<field name="code" />
+					<field name="name" />
+				</tree>
+			</field>
+		</record>
+
+		<record id="medical_patient_occupation_view_form" model="ir.ui.view">
+			<field name="name">medical.patient.occupation.form</field>
+			<field name="model">medical.patient.occupation</field>
+			<field name="priority" eval="8" />
+			<field name="arch" type="xml">
+				<form string="Patient Occupation">
+					<group>
+						<field name="code" />
+						<field name="name" />
+					</group>
+				</form>
+			</field>
+		</record>
+
+		<record id="medical_patient_occupation_view_search" model="ir.ui.view">
+			<field name="name">medical.patient.occupation.search</field>
+			<field name="model">medical.patient.occupation</field>
+			<field name="priority" eval="8" />
+			<field name="arch" type="xml">
+				<search string="Patient Occupation">
+					<field name="code" />
+					<field name="name" />
+					<newline />
+					<group expand="0" string="Group By...">
+						<filter string="Code" domain="[]" context="{'group_by':'code'}" />
+						<filter string="Name" domain="[]" context="{'group_by':'name'}" />
+					</group>
+				</search>
+			</field>
+		</record>
+
+		<record model="ir.actions.act_window" id="medical_patient_occupation_action">
+			<field name="name">Occupation</field>
+			<field name="type">ir.actions.act_window</field>
+			<field name="res_model">medical.patient.occupation</field>
+			<field name="view_type">form</field>
+			<field name="view_mode">tree,form</field>
+			<field name="search_view_id" ref="medical_patient_occupation_view_search" />
+			<field name="domain">[]</field>
+			<field name="context">{}</field>
+		</record>
+
+		<record model="ir.actions.act_window.view" id="medical_patient_occupation_action_form">
```

Comment:
Don't think it is an API change. I don't know why it was implemented like that. Only one required action with a pointer to the tree view and the search view. The form view is implicit.